src/averageAges.js

```diff
@@ -20,6 +20,14 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const menArray = people.filter(
+    person => person.sex === 'm'
```

Comment:
'm' is a magic string. It's a common antipattern. Consider using variables for this with meaningful names.